crypto/ex_data.c

```diff
@@ -287,8 +287,6 @@ int CRYPTO_dup_ex_data(int class_index, CRYPTO_EX_DATA *to,
         CRYPTOerr(CRYPTO_F_CRYPTO_DUP_EX_DATA, ERR_R_MALLOC_FAILURE);
         return 0;
     }
-    if (!CRYPTO_set_ex_data(to, mx - 1, NULL))
-        goto err;
```

Comment:
I mean in the loop all elements of "to" are overwritten
with values of "from", so if they are not zero there will be
a memory leak as well.

Do You suggest that the new ex_data already have some values?